Deeploy/Targets/Generic/Bindings.py

```diff
@@ -284,3 +304,36 @@
     NodeBinding(DequantChecker([PointerClass(int32_t)], [PointerClass(float32_t)]), DequantTemplate.referenceTemplate,
                 BasicTransformer),
 ]
+
+
+BasicBatchNormBindings = [
+    NodeBinding(
+        BatchNormChecker(
+            [PointerClass(float32_t), PointerClass(float32_t), PointerClass(float32_t), PointerClass(float32_t), PointerClass(float32_t)],
+            [PointerClass(float32_t)]
+        ),
+        BatchNormalizationTemplate.referenceTemplate,
+        BasicTransformer  # usa lo stesso se non hai un transformer dedicato
+    )
+]
+
+
+BasicConvTransposeBindings = [
+    NodeBinding(
+        ConvTransposeChecker(
+            [PointerClass(float32_t), PointerClass(float32_t), PointerClass(float32_t)],  # input, weight, bias
```

Comment:
Nice idea for these comments! It's easier to keep track of what each one represents